doc/developers/maintainer.rst

```diff
@@ -109,17 +114,20 @@ Making a release
    the RC releases. These changes should be made in master and cherry-picked
    into the release branch, only before the final release.
 
-   - Edit the doc/whats_new.rst file to add release title and commit
-     statistics. You can retrieve commit statistics with:
+   - Edit the ``doc/v0.99/whats_new.rst`` file to add release title and list of
+     contributors.
+     You can retrieve the list of contributor names with:
 
      ::
 
-       $ git shortlog -s 0.99.33.. | cut -f2- | sort --ignore-case | tr '\n' ';' | sed 's/;/, /g;s/, $//'
+       $ git shortlog -s 0.98.33.. | cut -f2- | sort --ignore-case | tr '\n' ';' | sed 's/;/, /g;s/, $//'
+
+     - For major releases, link the release highlights example from the ``doc/whats_new/v0.99.rst`` file.
 
    - Update the release date in ``whats_new.rst``
```

Comment:
I do not see a release date in the `whats_new.rst` file 😕.

Reply:
We forgot on some of them in the past.